nucypher/network/server.py

```diff
@@ -137,274 +137,285 @@ def __init__(self,
             _status_template_content = f.read()
         self._status_template = Template(_status_template_content)
 
-    def public_information(self):
-        """
-        REST endpoint for public keys and address..
-        """
-        headers = {'Content-Type': 'application/octet-stream'}
-        response = Response(
-            content=self._node_bytes_caster(),
-            headers=headers)
+        rest_app = self.rest_app = Flask("ursula-service")
 
-        return response
+        @rest_app.route("/public_information")
```

Comment:
Nice that it was that simple.
What confuses me is that all these views live inside `__init__`. Perhaps, better to have them being methods?